website/www/site/content/en/documentation/io/built-in/google-bigquery.md

```diff
@@ -602,59 +602,108 @@ as the previous example.
 BigQueryIO supports two methods of inserting data into BigQuery: load jobs and
 streaming inserts. Each insertion method provides different tradeoffs of cost,
 quota, and data consistency. See the BigQuery documentation for
-[load jobs](https://cloud.google.com/bigquery/loading-data) and
-[streaming inserts](https://cloud.google.com/bigquery/streaming-data-into-bigquery)
+[different data ingestion options](https://cloud.google.com/bigquery/loading-data)
+(specifically, [load jobs](https://cloud.google.com/bigquery/docs/batch-loading-data)
+and [streaming inserts](https://cloud.google.com/bigquery/streaming-data-into-bigquery))
 for more information about these tradeoffs.
 
+{{< paragraph class="language-java" >}}
 BigQueryIO chooses a default insertion method based on the input `PCollection`.
+You can use `withMethod` to specify the desired insertion method. See
+[`Write.Method`](https://beam.apache.org/releases/javadoc/{{< param release_latest >}}/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.Write.Method.html)
+for the list of the available methods and their restrictions.
+{{< /paragraph >}}
 
 {{< paragraph class="language-py" >}}
-BigQueryIO uses load jobs when you apply a BigQueryIO write transform to a
-bounded `PCollection`.
+BigQueryIO chooses a default insertion method based on the input `PCollection`.
+You can use `method` to specify the desired insertion method. See
+[`WriteToBigQuery`](https://beam.apache.org/releases/pydoc/{{< param release_latest >}}/apache_beam.io.gcp.bigquery.html#apache_beam.io.gcp.bigquery.WriteToBigQuery)
+for the list of the available methods and their restrictions.
 {{< /paragraph >}}
 
-{{< paragraph class="language-java" >}}
 BigQueryIO uses load jobs in the following situations:
-{{< /paragraph >}}
 
 {{< paragraph class="language-java" wrap="span" >}}
 * When you apply a BigQueryIO write transform to a bounded `PCollection`.
-* When you apply a BigQueryIO write transform to an unbounded `PCollection` and
```

Comment:
This doesn't seem to be the case based on me reading the code. Could you help double check?

https://github.com/apache/beam/blob/47cfbcb63f4d0642d26106485bc6fdb894da3086/sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java#L2474